wagtail_review/static_src/admin/src/components/Comments/style.scss

```diff
@@ -38,58 +37,44 @@
 
 .comment {
     display: block;
-    background: white;
     transition: top 0.5s ease 0s, right 0.5s ease 0s, height 0.5s ease 0s;
     font-size: 0.9em;
+    box-sizing: border-box;
+    padding-left: 13px !important;
```

Comment:
Why is the `!important` necessary here? Probably good to avoid if possible...

Reply:
I think this was me fighting against Wagtail's built-in styling.